### DISCUSSION TITLE
chiarimento R014

### DISCUSSION THREAD

**Discussion by OrazioCostanzo** [italia/ansc#878] [Supporto servizi cooperativi] [answered]
Salve, vorrei delle spiegazioni piu' dettagliate per quanto riguarda la R014

**Comment by mttfranci** [comment, marked as answer]
@OrazioCostanzo salve, il servizio serve solo a indicare la data di decorrenza dell'operazione ai fini anagrafici, qualora non sia stata indicata al momento della regitrazione dell'evento.

Saluti 

Assitenza ANSC
